native/src/routes/offers/containers/__tests__/OffersContainer.spec.js

```diff
@@ -48,8 +48,8 @@ describe('OffersContainer', () => {
       <OffersContainer navigation={navigation} route={route} />
     )
     expect(getByText('Offers')).toBeTruthy()
-    expect(() => getByText('loading')).toThrow('No instances found with text "loading"')
-    expect(() => getByText(errorText)).toThrow(`No instances found with text "${errorText}"`)
+    expect(() => getByText('loading')).toThrow('Unable to find an element with text: loading')
```

Comment:
I guess you could just remove the specific error message as well:
```suggestion
    expect(() => getByText('loading')).toThrow()
```

Reply:
or even better use: expect(() => queryByText('loading')).toBeFalsy().

Reply:
Or even better 🙈 `expect(queryByText('loading')).toBeFalsy()`

Reply:
done